intellij/org.eclipse.xtext.testlanguages.idea/META-INF/plugin.xml

```diff
@@ -0,0 +1,53 @@
+<idea-plugin version="2">
```

Comment:
I think this was pushed by accident.
The code generator produces bogus output for projects with more than one language. Idea does not support that so far.